styles.css

```diff
@@ -0,0 +1,107 @@
+
+* {
+  box-sizing: border-box;
+}
+
+#container {
+  /* width: 375px;
+  height: 710px; */
+  border: 1px solid #e5e5e5;
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
```

Comment:
- [ ] The container should fill all the size of the screen as represented in the Figma template , kindly Fix that 👍 
![Capture 4](https://user-images.githubusercontent.com/80458199/234552501-4c081afd-7a33-4f8a-8edf-9ecc1f9d55ef.JPG)